api/agent/call.go

```diff
@@ -178,22 +111,52 @@ func FromRequest(appName, path string, req *http.Request) CallOpt {
 	}
 }
 
-func noOverrideVars(key string) bool {
-	// descrepency in casing b/w req headers and env vars, force matches
-	return overrideVars[strings.ToUpper(key)]
+func buildEnv(id string, params Params, req *http.Request, app *models.App, route *models.Route) *models.CallEnv {
+	var env models.CallEnv
+	// we can base our new ones off our old ones, for speed
+	env.Header = map[string][]string(req.Header)
+
+	// TODO do we need to assert these all pass httplex.ValidHeaderFieldValue
+	for k, v := range app.Config {
+		env.SetBase(k, v)
+	}
+	for k, v := range route.Config {
+		env.SetBase(k, v)
+	}
+
+	env.SetBase("FN_FORMAT", route.Format)
+	env.SetBase("FN_APP_NAME", app.Name)
+	env.SetBase("FN_PATH", route.Path)
+	// TODO: might be a good idea to pass in: "FN_BASE_PATH" = fmt.Sprintf("/r/%s", appName) || "/" if using DNS entries per app
```

Comment:
Can you cover this TODO as well?

Reply:
i don't really agree with it, so no

Reply:
The only way a user could know this would be to parse the request URL, remove the FN_PATH part of it and figure it out on their own. A little cumbersome for basic information, but anyways, certainly not essential for this PR.